### PR TITLE
Added functions to serviceTest to utilize adopt and scratch project endpoints in Orch

### PR DESCRIPTION
**Pull request by ghost**
**PR checklist**
- [ ] For each library you've modified here, decide whether it requires a major, minor, or no version bump. (Click [here](/broadinstitute/workbench-libs/blob/develop/CONTRIBUTING.md) for guidance)

If you're doing a **major** or **minor** version bump to any libraries:
- [x] Bump the version in `project/Settings.scala` `createVersion()`
- [x] Update `CHANGELOG.md` for those libraries
- [x] I promise I used `@deprecated` instead of deleting code where possible

In all cases:
- [x] Replace the appropriate version hashes in `README.md` and the `CHANGELOG.md` for any libs you changed with `TRAVIS-REPLACE-ME` to auto-update the version string
- [ ] Get two thumbsworth of PR review
- [ ] Verify all tests go green (CI _and_ coverage tests)
- [ ] Squash commits and **merge to develop**
- [ ] Delete branch after merge

### CODE REVIEW

File: serviceTest/src/test/scala/org/broadinstitute/dsde/workbench/service/Orchestration.scala [hunk 1]
```diff
@@ -515,6 +515,30 @@ trait Orchestration extends RestClient with LazyLogging with SprayJsonSupport wi
...
+      logger.info(s"API post request: api/trial/manager/projects?operation=scratch&projectName=$projectName")
+      Try(postRequest(apiUrl(s"api/trial/manager/projects?operation=scratch&projectName=$projectName"))) match {
+        case Success(response) =>
+          implicit val impTrialProjectReport: RootJsonFormat[TrialProjectReport] = jsonFormat4(TrialProjectReport)
+          val trialProjectReport = response.parseJson.convertTo[TrialProjectReport]
+          logger.info(s"Scratched free tier project: ${trialProjectReport.name}")
+          trialProjectReport
+        case Failure(ex) => throw new Exception(s"Unable to scratch free tier project: $projectName", ex)
+      }
+    }
```
Comment (davidangb) [inline serviceTest/src/test/scala/org/broadinstitute/dsde/workbench/service/Orchestration.scala]: Do you need to wrap these in `Try`? Without the try, the underlying exception should bubble up. If you would rather keep the code as written so you can control the exception and return the nice error message, I support you.

**Review by davidangb — COMMENTED** [review]
Could you also mark `createTrialProjects` as deprecated in this PR? Doing it all in one PR would mean we don't need another hash update. I'm fine if you would rather do that in a future PR.

**Comment by davidangb** [comment]
also this is failing unit tests - you should be able to run those locally. I think you need a jsonformat for `UserStatusDetails` - see `reportTrialProjects` for an example.

**Comment by ghost** [comment]
Marked `createTrialProjects` as deprecated. Would you please check the wording and let me know if it could be improved?

**Review by davidangb — APPROVED** [review]
:+1: looks good. Deprecation wording is fine!

**Pull request merged**